packages/engine/test/services/variable-service.test.ts

```diff
@@ -17,29 +18,35 @@ executionState.insertStep(
   []
 );
 
+executionState.insertStep(
+  {
+    output: {
+      success: true,
+    }
+  },
+  "step_1",
+  []
+);
+
 describe('Variable Service', () => {
   test('Test resolve text with no variables', async () => {
     expect(await variableService.resolve('Hello world!', executionState)).toEqual(
       'Hello world!'
     );
   });
 
-  test('Test resolve configs variables', async () => {
-    expect(await variableService.resolve('${configs.msg}', executionState)).toEqual(
-      'Hello World'
-    );
-  });
-
-  test('Test resolve text with configs variables', async () => {
+  test('Test resolve text with double variables', async () => {
     expect(
-      await variableService.resolve('Price is ${configs.price}', executionState)
+      await variableService.resolve('Price is ${trigger.price}', executionState)
```

Comment:
Do these tests pass? Since the variable syntax was changed to `{{ }}` instead of `${ }`, right?

Also, It would be excellent if those tests can be added to the GitHub Actions pipeline.

Reply:
Sorry fixed, no they dont and added to github actions.